test/parallel/test-console-count.js

```diff
@@ -54,10 +54,12 @@ assert.strictEqual(buf, 'default: 2\n');
 
 process.stdout.write = stdoutWrite;
 
-// Symbol labels do not work
+// Symbol labels do not work. Only check that the `Error` is a `TypeError`. Do
+// not check the message because it is generated by the JavaScript engine and is
+// different in V8 and ChakraCore.
 assert.throws(
   () => console.count(Symbol('test')),
-  /^TypeError: Cannot convert a Symbol value to a string$/);
```

Comment:
@Trott, I know it's a ton of tedious work, but can these `assert.throws()` be modified to `common.expectsError()` for sake of consistency?

e.g. 

```js
common.expectsError(
  () => console.count(Symbol('test')),
  { type: TypeError }
);
```

Reply:
(feel free to say no for doing so in this PR :-) ...)

Reply:
Sounds like a great code-and-learn PR